test/recipe.e2e-spec.ts

```diff
@@ -132,10 +148,77 @@ describe('RecipeController (e2e)', () => {
     });
   });
 
+  describe('GET /recipes/user/:email', () => {
+    it('should fetch all users recipes', async () => {
+      const userPublicRecipe = createRecipe();
+      const userPrivateRecipe = createRecipe({ isPublic: false });
+      const otherUser = await prismaService.user.create({
+        data: createUser(),
+      });
+      const otherPublicRecipe = createRecipe();
+      const otherPrivateRecipe = createRecipe({ isPublic: false });
+      await prismaService.recipe.createMany({
+        data: [
+          { ...userPublicRecipe, authorId: user.id },
+          { ...userPrivateRecipe, authorId: user.id },
+          { ...otherPublicRecipe, authorId: otherUser.id },
+          { ...otherPrivateRecipe, authorId: otherUser.id },
+        ],
+      });
+      request(app.getHttpServer())
+        .get(`/recipes`)
+        .set({ Authorization: `Bearer ${accessToken}` })
+        .expect((response: request.Response) => {
+          expect(response.body.fetchedRecipes).toEqual(
```

Comment:
IMO, in e2e tests, you could use mocked service implementation (you do not have to make actual calls to the database)

Obviously the matter of choice.

Reply:
Btw, you have a test cases that already test if the response from db is the same as expected. Is there a reason to do it twice?
`                id: expect.any(Number),
                createdAt: expect.any(String),
                title: expect.any(String),
                description: expect.any(String),
                ingredients: expect.any(String),
                preparation: expect.any(String),
                isPublic: expect.any(Boolean),
                authorId: expect.any(user.id),`

Reply:
It's because before sending request, I create 2 recipes ( public and private ) for owner (let's call him that way) and 2 someone else's recipes ( public and private ).  Then the owner wants to get all recipes, so he should get all his as well as all other public recipes. So now in `expect.arrayContaining([...])` we check if it has any public recipe of someone else, and all of his